### ISSUE TITLE
Training details issues

### ISSUE THREAD

**Issue by Vocal-map** [XLearning-SCU/2022-CVPR-AirNet#16]
I am very interested in this paper, but I have developed some alienation while reading the code. May I ask how many 2080Ti gpus were used for training in this paper? And I found out in the paper that

> Set the maximum iteration number as 1500 To warm up, we first train CBDE by the optimizing Lcl for 100 iterations Then, we train the whole network with L for 1400 iterations The learning rate is initialized to 0.001 and then decreased to 0.0001 after 60 epochs After that, the learning rate is increased to half after each 125 epochs

Based on my reading of train.py and options.py, are the iterations here equivalent to epochs? It seems that the training needs to go through 1500epochs? And how did the batch size of this paper reach 400 * N? Can you please inform me of the training details? thanks

**Comment by XLearning-SCU** [comment]
Many thanks for your attention to our work. Your confusion may caused by the particularity of the All-in-One setting. In single degradation removal, epoch usually refer to train the model on the whole dataset one time. While, in the all-in-one setting of AirNet, the model will be trained on multiple degradation datasets, which have different size. Considering the AirNet will select one pair of images from each of the them, it may cause that some dataset have training many epochs while other haven't finish one epoch yet, which make confusion to decide which epoch is the epoch of the network. So we choose the epoch of BSD400 as the epoch of the network. Thus, in each epoch, we will have 400 iters, which contain N samples. In other words, after 400 iters, the BSD400 will train one epoch, which is the meaning of epoch in our work.
